src/subgraph/orders/index.ts

```diff
@@ -14,10 +14,13 @@ import { EMPTY_BYTES32, getUniqueValuesFromArray } from '../../common';
 export const getAllOrdersByUserAddress = async (
   chainId: Chain,
   userAddress: string,
+  subgraphEndpoint: string | undefined,
   count: number = 10,
 ): Promise<Order[]> => {
   try {
-    const subgraphEndpoint = getSubgraphEndpoint(chainId);
+    if(!subgraphEndpoint){
+      const subgraphEndpoint = getSubgraphEndpoint(chainId);
+    }
     const subgraphResponse = await request(subgraphEndpoint, fetchOrdersByUserQuery(userAddress, count));
```

Comment:
use the pagination query params `count` `skip`. check all request and take that on count

Reply:
Done